src/Computation/BitVecExprExtensions.cs

```diff
@@ -14,4 +14,9 @@ public static BigInteger AsConstant(this BitVecExpr self)
             ? ((BitVecNum) expr).BigInteger
             : throw new IrreducibleSymbolicExpressionException();
     }
+
+    public static BoolExpr AsBool(this BitVecExpr self, Context context)
+    {
+        return context.MkNot(context.MkEq(self, context.MkBV(0U, self.SortSize)));
+    }
 }
```

Comment:
As mentioned, I quite like the resulting syntax, but neither of these really need to be extension methods.